bip-0039.mediawiki

```diff
@@ -6,9 +6,9 @@
           Pavol Rusnak <stick@satoshilabs.com>
           Aaron Voisine <voisine@gmail.com>
           Sean Bowe <ewillbefull@gmail.com>
-  Comments-Summary: Unanimously Discourage for implementation
+  Comments-Summary: No comments yet.
```

Comment:
For context, the "Unanimously Discourage for implementation" assessment was added as part of a wider change in https://github.com/bitcoin/bips/pull/500. Seems fine to update if out of date / community agrees.

Reply:
Thanks for putting this in the title of your PR, alongside BIP39, unlike the last attempt to remove this language #1413

Reply:
@apoelstra thank you for the link to https://github.com/bitcoin/bips/pull/1413 for more context.

Reply:
> But I don't think you'll get "community agreement" on deleting this language entirely.

even tho I agree to what is stated in https://github.com/bitcoin/bips/pull/1413, I think the ship has already sailed and most of the wallets already implement it. "Unanimously Discourage for implementation" was therefore pointless back then and even more now.

If necessary, it would be better to add a `Shortcomings`(or similar) section to this BIP and "copy" the comment from https://github.com/bitcoin/bips/pull/1413 which at least give reader some useful info, instead of just  fuding with "Unanimously Discourage for implementation"

Reply:
@scgbckbone I've edited my comments above. It may be a good idea to drop this here and discuss a change to the comment summary in a separate PR, and/or add relevant expert review to the wiki.

(Also, as described in BIP 2: "To avoid doubt: comments and status are unrelated metrics to judge a BIP, and neither should be directly influencing the other." Thus, no need to do them both here.)

Reply:
While I agree that the comments feature of the existing BIP Process has failed to be meaningfully adopted, I don’t think this change is in line with the currently active process. I would agree with BIP 39 being moved to Final, but I would suggest that you defer the change regarding the comment summary until a new BIP Process is adopted that supports the removal (as e.g. the one that I have been working on would).